src/components/statusFilter.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+import { twJoin } from "tailwind-merge";
+
+interface OptionStyle {
+  option: string;
+  status: string;
+  activeClass: string;
+  inactiveClass: string;
+}
```

Comment:
I suggest you remove property inactiveClass because of comment below. Also, I suggest changing the property option to something more meaningful, like label (as it is going to be the text inside the button). Another thing: change the string types of label (option) and status to a Union Type.
```suggestion
type Status = 'ACTIVE' | 'INACTIVE' | 'ALL'

type OptionLabel = 'ATIVOS' | 'INATIVOS' | 'TODOS'

interface OptionData {
  label: OptionLabel;
  status: Status;
  activeClass: string;
}
```

Reply:
we also did that